applications/openshift/api-server/api_server_no_adm_ctrl_plugins_disabled/rule.yml

```diff
@@ -0,0 +1,45 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Check that no admission control plugins are disabled'
+
+description: |-
+    To make sure none of them is explicitly enabled, call
+    <pre>$ oc -n openshift-kube-apiserver get configmap config -o json | jq -r '.data."config.yaml"' | jq '.apiServerArguments."disable-admission-plugins"'</pre>
+
+rationale: |-
+    Several hardening controls depend on certain API server admission plugins
+    being enabled. 
```

Comment:
That's great, but rationale should be flushed out more.

Reply:
what about now?